api/v1alpha1/ndb_api_helpers.go

```diff
@@ -186,19 +186,32 @@ func GetOOBProfiles(ctx context.Context, ndbclient *ndbclient.NDBClient, dbType
 
 func GetDatabaseEngineName(dbType string) string {
 	switch dbType {
-	case "postgres":
+	case DATABASE_TYPE_POSTGRES:
 		return DATABASE_ENGINE_TYPE_POSTGRES
-	case "mysql":
+	case DATABASE_TYPE_MYSQL:
 		return DATABASE_ENGINE_TYPE_MYSQL
-	case "mongodb":
+	case DATABASE_TYPE_MONGODB:
 		return DATABASE_ENGINE_TYPE_MONGODB
-	case "generic":
+	case DATABASE_TYPE_GENERIC:
 		return DATABASE_ENGINE_TYPE_GENERIC
 	default:
 		return ""
 	}
 }
 
+func GetDatabasePortByType(dbType string) int32 {
+	switch dbType {
+	case DATABASE_TYPE_POSTGRES:
+		return DATABASE_DEFAULT_PORT_POSTGRES
+	case DATABASE_TYPE_MONGODB:
+		return DATABASE_DEFAULT_PORT_MONGODB
+	case DATABASE_TYPE_MYSQL:
+		return DATABASE_DEFAULT_PORT_MYSQL
+	default:
+		return 80
```

Comment:
why not 443?

Reply:
This could ideally be anything, I used 80 as a placeholder for the default block. Will not reach a condition where the 'default' port will be used.